### PR TITLE
Updated Cubeviz Displaying Spectra

### PR DESCRIPTION
**Pull request by PatrickOgle**
Added documentation to Cubeviz:Displaying Spectra

### CODE REVIEW

File: docs/cubeviz/displayspectra.rst [hunk 1]
```diff
@@ -2,20 +2,14 @@
...
-Defining Spectral Regions
-=========================
-
-Coming soon
-
-Plot Settings
-=============
-
-Coming soon
+    `Gaussian Smooth <https://jdaviz.readthedocs.io/en/latest/specviz/displaying.html`_
```
Comment (rosteen) [inline docs/cubeviz/displayspectra.rst:14]:
```suggestion
    `Displaying Spectra (Specviz) <https://jdaviz.readthedocs.io/en/latest/specviz/displaying.html>`_
```
Fix missing close bracket, change the text to match the linked page.

File: docs/cubeviz/displayspectra.rst [hunk 2]
```diff
@@ -2,20 +2,14 @@
...
+        Specviz documentation on gaussian smoothing of 1D spectra.
```
Comment (rosteen) [inline docs/cubeviz/displayspectra.rst:15]:
```suggestion
        Specviz documentation on displaying 1D spectra.
```

**Review by rosteen — CHANGES_REQUESTED** [review]
The text in the `seealso` link didn't seem to match the linked page, so I made a couple suggestions.

File: docs/cubeviz/displayspectra.rst [hunk 3]
```diff
@@ -2,20 +2,14 @@
...
-=========================
-
-Coming soon
-
-Plot Settings
-=============
-
-Coming soon
+    `Displaying Spectra (Specviz) <https://jdaviz.readthedocs.io/en/latest/specviz/displaying.html>`_
+        Specviz documentation on gaussian smoothing of 1D spectra.
```
Comment (PatrickOgle) [inline docs/cubeviz/displayspectra.rst:15]:
```suggestion
        Documentation on displaying spectra in a 1D spectrum viewer.
```

**Review by rosteen — APPROVED** [review]

File: docs/cubeviz/displayspectra.rst [hunk 4]
```diff
@@ -2,20 +2,14 @@
 Displaying Spectra
 ******************
 
-Coming soon
+A collapsed spectrum of the cube displayed in the upper-left viewer
+automatically appears in the 1D spectrum viewer, using the Maximum
+collapse method.  The collapse method can be changed in the Viewer
+tab of the :guilabel:`Gear` icon in the spectrum viewer. Additional spectra
+can be loaded into the spectrum viewer, as detailed in the linked documentation
+below.
```
Comment (PatrickOgle) [inline docs/cubeviz/displayspectra.rst:10]:
```suggestion
below. 
```

**Review by javerbukh — APPROVED** [review]
Looks good!

**Comment by javerbukh** [comment]
Merging.

**Pull request merged**